core/resource/handler/v1beta1/resource.go

```diff
@@ -434,10 +434,7 @@ func (rh ResourceHandler) GetResourceChangelogs(ctx context.Context, req *pb.Get
 		return nil, errors.GRPCErr(err, "invalid project name")
 	}
 
-	resourceName, err := resource.NameFrom(req.GetResourceName())
-	if err != nil {
-		return nil, errors.GRPCErr(err, "invalid resource name")
-	}
+	resourceName := resource.Name(req.GetResourceName())
```

Comment:
We have removed the validation for empty name. We should add that somewhere.

Reply:
got it, adding